### PR TITLE
chore: clear confusion in relation tables(#3591, #4114)

### PR DESCRIPTION
**Pull request by ankur-arch**
- [x] Re-arrange the relation heading
- [x] Move content to explicit m-n section
- [x] Reword many to many to m-n for consistency

closes ([#3591](https://github.com/prisma/docs/issues/3591), [#4114](https://github.com/prisma/docs/issues/4114))

### CODE REVIEW

**Review by nikolasburk — APPROVED** [review]

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 1]
```diff
@@ -83,13 +82,17 @@ ALTER TABLE "CategoriesOnPosts" ADD CONSTRAINT "CategoriesOnPosts_postId_fkey" F
 ALTER TABLE "CategoriesOnPosts" ADD CONSTRAINT "CategoriesOnPosts_categoryId_fkey" FOREIGN KEY ("categoryId") REFERENCES "Category"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
 ```
 
+Note that the same rules as for [1-n-relations](one-to-many-relations) apply (because `Post`↔ `CategoriesOnPosts` and `Category` ↔ `CategoriesOnPosts` are both in fact 1-n-relations), which means one side of the relation needs to be annotated with the `@relation` attribute.
```
Comment (jharrell) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
```suggestion
Note that the same rules as for [1-n relations](one-to-many-relations) apply (because `Post`↔ `CategoriesOnPosts` and `Category` ↔ `CategoriesOnPosts` are both in fact 1-n relations), which means one side of the relation needs to be annotated with the `@relation` attribute.
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 2]
```diff
@@ -83,13 +82,17 @@ ALTER TABLE "CategoriesOnPosts" ADD CONSTRAINT "CategoriesOnPosts_postId_fkey" F
...
+
 #### Querying an explicit many-to-many
 
-The following section demonstrates how to query an explicit many-to-many relation. You can query the relation model directly (`prisma.categoriesOnPosts(...)`), or use nested queries to go from `Post` -> `CategoriesOnPosts` -> `Category` or the other way.
+The following section demonstrates how to query an explicit m-n relation. You can query the relation model directly (`prisma.categoriesOnPosts(...)`), or use nested queries to go from `Post` -> `CategoriesOnPosts` -> `Category` or the other way.
 
 The following query does three things:
 
-1. Creates a `Post`
+1. Creates a `Post` "assignedBy" TEXT NOT NULL,
```
Comment (jharrell) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Is this a mistake? Feels out of place.

Comment (Druue) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: > "assignedBy" TEXT NOT NULL

Is this supposed to be here? It's not immediately clear what the goal of this is to me

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 3]
```diff
@@ -423,54 +425,20 @@ model Category {
...
-  assignedAt DateTime @default(now())
-
-  @@id([postId, categoryId])
-}
-```
-
-In this example, the `assignedAt` field adds additional information about the relation between `Post` and `Category`: it specifies when the post was assigned to the category.
-
-Note that the same rules as for [1-n-relations](one-to-many-relations) apply (because `Post`↔ `CategoriesOnPosts` and `Category` ↔ `CategoriesOnPosts` are both in fact 1-n-relations), which means one side of the relation needs to be annotated with the `@relation` attribute.
+A relation table (also sometimes called _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modelling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
```
Comment (Druue) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
```suggestion
A relation table (also sometimes called a _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modelling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 4]
```diff
@@ -423,54 +425,20 @@ model Category {
...
 
-When you don't need to attach additional information to the relation, you can model m-n-relations as [implicit many-to-many relations](#implicit-many-to-many-relations). If you're not using Prisma Migrate but obtain your data model from [introspection](/concepts/components/introspection), you can still make use of implicit many-to-many relations by following Prisma's [conventions for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations).
+We recommend using [implicit](#implicit-many-to-many-relations) m-n relations, where Prisma generates the relation table in the underlying database. [Explicit](#explicit-many-to-many-relations) m-n relations should be used when you need to store additional data in the relations, such as the date the relation was created.
```
Comment (jharrell) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Is this right? Does Prisma generate a table in the DB? I was under the impression that implicit m-n is handled at the engine/client level.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 2 above)
Comment (jharrell) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: @Druue beat me to it. Resolving 🙏

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 4 above)
Comment (Druue) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]:
```suggestion
We recommend using [implicit](#implicit-many-to-many-relations) m-n relations. This is where Prisma automatically generates the relation table in the underlying database. [Explicit](#explicit-many-to-many-relations) m-n relations should be used when you need to store additional data in the relations, such as the date the relation was created.
```

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 5]
```diff
@@ -423,54 +425,20 @@ model Category {
...
-
-Here is an example for a relation table called `CategoriesOnPosts`:
-
-```prisma
-model Post {
-  id         Int                 @id @default(autoincrement())
-  title      String
-  categories CategoriesOnPosts[]
-}
+## Relation tables
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Why this change in headline level? Now it is on the same level as "Relational databases" and "MongoDB", which does not seem correct.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 4 above)
Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Prisma creates the table during migrations. We still need a way to store the "foreign keys".

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 5 above)
Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: The intention was to separate it out as a concept to act as a glossary item. Rather than a different type of relation strategy.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 2 above)
Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Clearing it out in the next commit.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 5 above)
Comment (janpio) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: This page defines how relations work in Prisma on either relational databases, or MongoDB. I don't think "Relational tables" (which are only relevant for relational databases) can live on the same level as these.

Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Hmm, I agree. I'll move it to h3 again, the content should be clear enough

Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: fixed in latest commit

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 6]
```diff
@@ -14,14 +14,14 @@ Prisma schema syntax and the implementation in the underlying database differs b
...
+In relational databases, m-n relations are typically modelled via [relation tables](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables). m-n relations can be either [explicit](#explicit-many-to-many-relations) or [implicit](#implicit-many-to-many-relations) in the Prisma schema. We recommend using [implicit](#implicit-many-to-many-relations) m-n relations if you do not need to store any additional meta-data in the relation table itself. You can always migrate to an [explicit](#explict-many-to-many-relations) m-n relation later if needed.
 
 ### Explicit many-to-many relations
 
-In an explicit many-to-many relation, the **relation table is represented as a model in the Prisma schema** and can be used in queries. Explicit many-to-many relations define three models:
+In an explicit m-n relation, the **relation table is represented as a model in the Prisma schema** and can be used in queries. Explicit m-n relations define three models:
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx:21]: Change in our style guide? I remember not long ago this was expanded.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx [hunk 7]
```diff
@@ -14,14 +14,14 @@ Prisma schema syntax and the implementation in the underlying database differs b
 
 ## Relational databases
 
-In relational databases, m-n-relations are typically modelled via [relation tables](/concepts/components/prisma-schema/relations/many-to-many-relations#relation-tables). m-n-relations can be either [explicit](#explicit-many-to-many-relations) or [implicit](#implicit-many-to-many-relations) in the Prisma schema.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx:17]: Change in our style guide from `m-n-relations` to `m-n relations`?

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 6 above)
Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx:21]: wanted to keep consistency as in the section heading we [mention](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#relational-databases) m-n relations

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 7 above)
Comment (ankur-arch) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx:17]: adding the hyphen from our style guide.

File: content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx (hunk 3 above)
Comment (Druue) [reply content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx]: Is there something that I'm missing? This is grammatically incorrect without either the 'a' or modifying 'table' to be plural
